### PR TITLE
Labelled fields

### PR DESCRIPTION
**Pull request by lpil**

### CODE REVIEW

File: concepts/labelled-fields/about.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+```gleam
+pub type Rectangle {
+  Rectangle(
+    width: Float,
+    height: Float,
+  )
+}
+```
+
+Labelled can be used to give arguments in any order to the constructor of a record.
```
Comment (giacomocavalieri) [inline concepts/labelled-fields/about.md:25]:
```suggestion
Labels can be used to give arguments in any order to the constructor of a record.
```

File: concepts/labelled-fields/introduction.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+```gleam
+pub type Rectangle {
+  Rectangle(
+    width: Float,
+    height: Float,
+  )
+}
+```
+
+Labelled can be used to give arguments in any order to the constructor of a record.
```
Comment (giacomocavalieri) [inline concepts/labelled-fields/introduction.md:25]:
```suggestion
Labels can be used to give arguments in any order to the constructor of a record.
```

File: exercises/concept/bandwagoner/.docs/hints.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+- The record update syntax can be used to create a copy of a record but with one or more fields having a new value.
+
+## 6. Check for same team
+
+- Records have built-in structural equality, which means that records that have the same values are equal.
+
+## 7. Check if you should root for a team
+
+- The best way to execute logic based on the team's value is to use a case expression.
+- If you want to add an additional condition to a pattern, you can add a guards.
```
Comment (giacomocavalieri) [inline exercises/concept/bandwagoner/.docs/hints.md:14]: maybe "a guard" or "guards"?

File: exercises/concept/bandwagoner/.docs/introduction.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+```gleam
+pub type Rectangle {
+  Rectangle(
+    width: Float,
+    height: Float,
+  )
+}
+```
+
+Labelled can be used to give arguments in any order to the constructor of a record.
```
Comment (giacomocavalieri) [inline exercises/concept/bandwagoner/.docs/introduction.md:27]:
```suggestion
Labels can be used to give arguments in any order to the constructor of a record.
```

File: exercises/concept/bandwagoner/.meta/design.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Accessors for multiple variant custom types.
+
+## Concepts
+
+- `labelled-fields`
+
+## Prerequisites
+
+- `basics`: know what and int and a string is.
```
Comment (giacomocavalieri) [inline exercises/concept/bandwagoner/.meta/design.md:22]:
```suggestion
- `basics`: know what an int and a string is.
```

**Comment by lpil** [comment]
Thank you once again @giacomocavalieri !! 💜

**Pull request merged**